### ISSUE TITLE
[ACTIVE_ARBITRATED] SWARM-CLAIM D04|PR835|UNIQUE-LOSS-EXPOSURE-GUARD|CURRENT-MAIN-REPAIR|V1

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1260]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1260
INSTANCE_TAG: SOL-SCI-CLOSER-20260911T1220Z
SWARM_LANE_KEY: D04|PR835|UNIQUE-LOSS-EXPOSURE-GUARD|CURRENT-MAIN-REPAIR|V1
PREFERRED_LANE: #2
PREFERRED_WORK_KIND: PERFORMANCE_RUNTIME
SELECTED_WORK_KIND: REDTEAM_AUDIT -> DEFECT_REPAIR_IF_CONFIRMED
PARENT_ISSUES: #5 #548 #723 #835 #856 #867 #990
STATUS: ACTIVE_ARBITRATED
LIVE_MAIN_AT_CLAIM: 7accc766d76693a69dabb5ff853cb35040ec4728
CLAIMED_AT_UTC: 2026-09-11T12:29:53Z
LEASE_UNTIL_UTC: 2026-09-11T18:29:53Z

POSTCLAIM ARBITRATION:
- direct newest-first OPEN issue collection after claim shows #1260 as newest issue and no older exact-semantic D04 PR835 exposure-guard repair owner;
- exact OPEN semantic search `UNIQUE-LOSS-EXPOSURE-GUARD` returns #1260 only for this seam; other hits are readiness/pilot/D10 consumers that explicitly avoid D04 Product mutation;
- #1248 pilot consumes D04 exposure guards but explicitly does not own #835 semantics;
- #1251 readiness consumes ledger identity/count but explicitly avoids D04 Product mutation;
- #1164/#1043 D10 durable launch-input repair is a downstream verifier lineage, not canonical #835 guard mutation;
- arbitration winner: SWARM-1260. One Product branch/PR maximum.

ROUTING / CANDIDATE COMPARISON:
1. Current-main D04 unique-loss/exposure guard trust binding — SELECTED. Exact anti-replay/exposure accounting directly gates optimizer step 1; merged two-clean proof remediation #990 is already terminal, so this is the next distinct arithmetic/trust seam rather than duplicate packing infrastructure.
2. PR #1041 decontam->G05/G06 repair/execution — YIELD to active #1250.
3. PR #945 balance / audit — YIELD to #1252/#1253.
4. Balanced split/readiness/recovery/bounded pilot — YIELD to #1246/#1251/#1233/#1248.
5. New source families — REJECT now as farther from first legal optimizer step.

CONCRETE CURRENT-MAIN FINDING TO PROVE/FALSIFY:
`src/twelve_six/data/unique_loss_ledger_v2.py::ExposureReplayGuard.__init__` checks only `schema_version == LEDGER_SCHEMA`, then directly trusts the supplied `ledger_identity_sha256`, materialization/packing identities, `one_pass_unique_nonignored_causal_loss_positions` and segment identities/lengths. It does not recompute the ledger self-hash, enforce exact ledger keys, or call `verify_ledger(materialization, ledger)`. Therefore a caller can plausibly keep a stale externally trusted ledger identity while mutating one-pass capacity and/or segment lengths, then construct a guard with an inflated authorized budget. Existing tests prove `verify_ledger()` rejects aggregate tamper, but do not require the guard constructor itself to consume verified ledger bytes.

OWNED SURFACE:
- existing `src/twelve_six/data/unique_loss_ledger_v2.py` only;
- focused/adversarial `tests/test_unique_loss_ledger_v2.py`;
- minimal existing consumers only if exact compatibility demands an externally expected ledger identity argument; no new ledger/packing/replay architecture.

AVOID:
- no tokenizer/packer/postpack algorithm changes; no #856/#990 Product mutation;
- no D03 corpus/decontam/privacy/balance/split mutation;
- no D05/D06/readiness/recipe/pilot Product mutation;
- no corpus payload execution, tokenizer fit, optimizer/model update, final-test access, paid compute, foreign pretrained weights, external LLM/API corpus intelligence.

ACCEPTANCE:
- reproduce/falsify stale-identity ledger mutation against ExposureReplayGuard;
- if confirmed, guard creation fails closed on stale identity, semantic mutation, unknown schema extension and capacity/segment drift before any position can be authorized;
- preserve a caller path to independently expected exact ledger identity rather than deriving trust solely from packet self-consistency;
- bool/int aliases, duplicate segment identities, replay/overlap, oversubscription and resume accounting remain fail closed;
- preserve deterministic build/verify APIs and existing valid callers with the smallest repair;
- focused/adversarial LOCAL_FREE tests + exact-head shared CI terminal SUCCESS required before fresh independent audit; producer does not self-certify or merge.

TRUTH_BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
OPTIMIZER_UPDATES_EXECUTED=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
SCOUT FINDING FALSIFIED / PRODUCT MUTATION ABANDONED — 2026-09-11T12:33Z

The initial low-level suspicion is **not** a valid current canonical Product blocker. Current main already contains `src/twelve_six/data/identity_safe_exposure_guard.py`, where `IdentitySafeExposureReplayGuard` requires an externally supplied `expected_ledger_identity_sha256`, exact closed-world V2 ledger keys, exact observed-vs-expected identity equality, and recomputed ledger self-hash before delegating to `ExposureReplayGuard`.

Canonical `deterministic_exposure_order.py` imports/uses this identity-safe guard and `require_expected_ledger_identity` for preflight. Existing focused regressions already prove:
- stale identity + mutated segment capacity => `self-hash mismatch`;
- coherently rehashed substituted ledger => rejected against frozen external stage identity;
- unknown self-hashed ledger field => rejected by closed-world schema.

Therefore the base-class constructor alone is not sufficient evidence of a launch fail-open; patching it would duplicate/perturb an already hardened incumbent boundary. No Product file was changed and no PR was opened. The just-created branch `d04/unique-loss-exposure-guard-binding-1260` remains at exact main with zero commits and is not a candidate.

STATUS: RELEASED_FALSE_POSITIVE / ROTATING.

Truth boundary unchanged: optimized-target exposure=0; optimizer updates=0; training=false; learned weights=false; final test untouched; paid compute=false.
